### PR TITLE
Add support for User Groups

### PR DESCRIPTION
**Pull request by mirzal**
Add basic support for User Groups functionality in AirGun.

This covers #88

### CODE REVIEW

File: airgun/entities/usergroup.py [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.usergroup import UserGroupCreateView, UserGroupDetailsView, UserGroupsView
+
+
+class UserGroupEntity(BaseEntity):
+
+    def create(self, values):
+        view = self.navigate_to(self, 'New')
+        view.fill(values)
+        self.browser.selenium.implicitly_wait(3)
```
Comment (oshtaier) [inline airgun/entities/usergroup.py]: not a chance that we use implicit waits in our code, please find alternative solutions (e.g. wait_for library or any other visibility validations)

Comment (mirzal) [reply airgun/entities/usergroup.py]: Oh, that's embarrassing. I have used it during debugging sessions and never meant to actually commit it, but forgot to delete it. Good catch!

File: airgun/entities/usergroup.py [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+class EditUser(NavigateStep):
+    VIEW = UserGroupDetailsView
+
+    def prerequisite(self, *args, **kwargs):
+        return self.navigate_to(self.obj, 'All')
+
+    def step(self, *args, **kwargs):
+        entity_name = kwargs.get('entity_name')
+        self.parent.search(entity_name)
+        self.parent.table.row(username=entity_name)['Username'].widget.click()
```
Comment (oshtaier) [inline airgun/entities/usergroup.py]: `['Username']`? or `['Name']`?

File: airgun/entities/usergroup.py [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+    VIEW = UserGroupCreateView
+
+    prerequisite = NavigateToSibling('All')
+
+    def step(self, *args, **kwargs):
+        self.parent.new.click()
+
+
+@navigator.register(UserGroupEntity, 'Edit')
+class EditUser(NavigateStep):
```
Comment (oshtaier) [inline airgun/entities/usergroup.py]: `EditUser` ?

File: airgun/entities/usergroup.py [hunk 4]
```diff
@@ -0,0 +1,64 @@
+from navmazing import NavigateToSibling
+
+from airgun.entities.base import BaseEntity
+from airgun.navigation import NavigateStep, navigator
+from airgun.views.usergroup import (UserGroupCreateView,
+                                    UserGroupDetailsView, UserGroupsView)
```
Comment (oshtaier) [inline airgun/entities/usergroup.py]:
```
+from airgun.views.usergroup import (
    UserGroupCreateView,
    UserGroupDetailsView, 
    UserGroupsView
)
```

File: airgun/entities/usergroup.py (hunk 3 above)
Comment (mirzal) [reply airgun/entities/usergroup.py]: `EditUserGroup` of course. I used `entities/user.py` as example.

File: airgun/entities/usergroup.py (hunk 2 above)
Comment (mirzal) [reply airgun/entities/usergroup.py]: `['Name']` and `name` of course.

File: airgun/entities/usergroup.py (hunk 4 above)
Comment (mirzal) [reply airgun/entities/usergroup.py]: Done

**Comment by oshtaier** [comment]
Add sanity tests in robottelo and run tests. Then I think we ready to merge

**Pull request merged**